Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM docker.io/library/node:15 as builder
+FROM docker.io/library/node:16 as builder
```

Comment:
It depends on what we are trying to achieve. If we keep `node:16` then docker will use the latest image version. Otherwise the version will be constant (e.g. `16.15.1`).

Reply:
I guess the point here is to use a fixed, reproducable version of node for the built image, but to allow minor updates and such in the container used to do the building?

I've no strong opinion either way; I'd probably lean to using 16.15.1 in both places to keep both stages as reproducable as possible (with the tradeoff of not getting security updates to either), but shrug!